go.mod

```diff
@@ -5,6 +5,9 @@ go 1.13
 require (
 	github.com/DataDog/zstd v1.4.5 // indirect
 	github.com/beevik/ntp v0.3.0
+	github.com/chzyer/logex v1.1.10 // indirect
```

Comment:
@volknanebo What are these 3 dependencies from `github.com/chzyer` used for?

Reply:
DOSKEY like CLI history .

Reply:
Ok, clear now, thanks.